### PR TITLE
Add Minestom to native platforms list

### PR DESCRIPTION
**Pull request by kezz**
Following the merge of https://github.com/Minestom/Minestom/pull/157, Minestom now provides native support for Adventure.

### CODE REVIEW

File: source/platform/native.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
+As of this writing, Velocity 1.1.0+, Sponge 8+, Paper 1.16.5+ and Minestom support Adventure
```
Comment (Ehhthan) [inline source/platform/native.rst:7]: Should this include an oxford comma?

Comment (Electroid) [reply source/platform/native.rst:7]: Yes

Comment (kezz) [reply source/platform/native.rst:7]: Just letting my hatred of Oxford commas get the better of me again...

File: source/platform/native.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
+As of this writing, Velocity 1.1.0+, Sponge 8+, Paper 1.16.5+, and Minestom support Adventure
```
Comment (kashike) [inline source/platform/native.rst:7]: It might be better to turn this into a table at this point - "Platform Name" and "Minimum Version" columns.

**Review by kashike — CHANGES_REQUESTED** [review]

**Comment by kezz** [comment]
Moved the information into a table - it's worth noting that the table doesn't seem to wrap text if it's oversized. I'm not sure if this is something that needs to be done manually or if I'm missing something about how tables work in RST. Not an issue with the table at the moment but it might be something to keep an eye on in the future.

File: source/platform/native.rst [hunk 3]
```diff
@@ -4,21 +4,16 @@
...
-
-As of Paper 1.16.5 build 473, Paper now natively supports all Adventure functionality.
+The following software provide native support for Adventure.
+
++----------+-----------------------+-----------------------------------------------------------------------------------------------+
+| Platform | Minimum Version       | Additional Notes                                                                              |
++==========+=======================+===============================================================================================+
+| Sponge   | Sponge 8              |                                                                                               |
++----------+-----------------------+-----------------------------------------------------------------------------------------------+
+| Velocity | 1.1.0 build 158       | Only sessages, action bars, boss bars and titles are supported                                |
```
Comment (kashike) [inline source/platform/native.rst:14]:
```suggestion
| Velocity | 1.1.0 build 158       | Only messages, action bars, boss bars, and titles are supported                                |
```

**Review by kashike — CHANGES_REQUESTED** [review]

**Review by kashike — APPROVED** [review]

**Pull request merged**